thingif/src/main/java/com/kii/thingif/clause/trigger/EqualsClauseInTrigger.java

```diff
@@ -13,6 +13,8 @@
     private @NonNull Object value;
     private @NonNull String alias;
 
+    private volatile int hashCode; // cached hashcode for performance
```

Comment:
> Do we need to makehashCode thread safe?
> If hashCode is not thread safe, I think big problem does not happen.

If we would like to cache hashCode, `volatile` is suggested to provide(mentioned in `Effective Java`). I think in Android app, multithread is comm, isn't it?